src/components/MediaSettings/MediaSettings.vue

```diff
@@ -119,6 +119,17 @@
 				:token="token"
 				@update-background="handleUpdateVirtualBackground" />
 
+			<!-- Setting guest name -->
+			<template v-if="isUnknownGuest">
+				<label for="guestname-prompt" class="media-settings__guestname-label">
+					{{ t('spreed','Please enter a display name') }}
```

Comment:
```suggestion
					{{ t('spreed','Enter your name') }}
```

Same wording as in the normal dialog?